contracts/SnappAuction.sol

```diff
@@ -8,6 +8,19 @@ contract SnappAuction is SnappBase {
     uint16 public constant AUCTION_BATCH_SIZE = 1000;
     uint8 public constant AUCTION_RESERVED_ACCOUNTS = 50;
     uint8 public constant AUCTION_RESERVED_ACCOUNT_BATCH_SIZE = 10;
+    
+    struct StandingOrderBatch {
+        bytes32 orderHash;
+        uint validFromIndex; // validity is inclusive of the auction index
+        // validToIndex is indirectly given as validFromIndex of next Orderbatch
+    }
+
+    struct StandingOrderData {
+        mapping(uint => StandingOrderBatch) reservedAccountOrders;
+        uint currentBatchIndex;
+    }
+
+    mapping (uint16 => StandingOrderData) public standingOrderData;
```

Comment:
My only worry here is using `standingOrderData` and `StandingOrderData` for two different but similar things. I would propose either that 
- `StandingOrderData` become `StandingOrder` or
- `standingOrderData` become `standingOrders` or `standingOrderMap`